uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -34,6 +34,7 @@ define([
       this.absenceTypes = [];
       this.calendar = {};
       this.contactName = null;
+      this.didSubmit = false;
```

Comment:
The name of this property is misleading, as by this name you would expect to always stay `true` after the user has clicked "submit" (instead, it can go back to `false` if some error happens)

Probably a better name is `submitting`